.github/workflows/main.yml

```diff
@@ -349,14 +302,16 @@ jobs:
     - run: |
         cargo build -p wasm-bindgen-cli
         ln -snf `pwd`/target/debug/wasm-bindgen $(dirname `which cargo`)/wasm-bindgen
-    - run: mv _package.json package.json && npm install && rm package.json
+    - run: mv _package.json package.json && corepack pnpm install && rm package.json
     - run: |
-        for dir in `ls examples | grep -v README | grep -v raytrace | grep -v deno | grep -v wasm-audio-worklet`; do
-          (cd examples/$dir &&
-            (npm run build -- --output-path ../../exbuild/$dir ||
-              (./build.sh && mkdir -p ../../exbuild/$dir && cp -r ./* ../../exbuild/$dir && rm ../../exbuild/$dir/**/.gitignore))
+        function build() {
+          (cd examples/$1 &&
+            (corepack pnpm run build -- --output-path ../../exbuild/$1 ||
+              (./build.sh && mkdir -p ../../exbuild/$1 && cp -r ./* ../../exbuild/$1 && rm ../../exbuild/$1/**/.gitignore))
           ) || exit 1;
-        done
+        }
+        export -f build
+        ls examples | grep -v README | grep -v raytrace | grep -v deno | grep -v wasm-audio-worklet | parallel -j4 build
```

Comment:
As mentioned in the other PR, more parallelization doesn't help much because Cargo just gets blocked by other Cargo instances. I haven't fine-tuned the parallelization but `-j8` was slower than `-j4`.

Reply:
I think `-j4` saved about 20 to 30 seconds, but I'll compare directly.

Otherwise I think it's more readable as a function even if we remove the `parallel` for now.

Reply:
> I think this just needs to be split up between Cargo and Webpack runs.

It looks like most of that happens through `wasm-pack` so that's not that simple.